src/gsi/m_berror_stats_reg.f90

```diff
@@ -208,6 +209,17 @@ subroutine berror_read_bal_reg(msig,mlat,agvi,bvi,wgvi,mype,unit)
        rlsigo(k)=log(sigma_avn(k))
     enddo
 
+!  compute delta sigma of global level
+    if(usenewgfsberror)then
+      allocate(dlsig_glb(msig))
+      dlsig_glb(1)=log(sigma_avn(1))-log(sigma_avn(2))
+      do k=2,msig-1
+         dlsig_glb(k)=half*(log(sigma_avn(k-1))-log(sigma_avn(k)))
```

Comment:
Shouldn't this be:
half*(log(sigma_avn(k-1))-log(sigma_avn(**k+1**)))
instead of
half*(log(sigma_avn(k-1))-log(sigma_avn(k)))

Reply:
Yes, you are right.  Should be  half*(log(sigma_avn(k-1))-log(sigma_avn(**k+1**))).
I will fix it. Thanks!

Reply:
This has been fixed and push back to master. Please review again! Thanks!